dags/merino_jobs.py

```diff
@@ -72,14 +73,37 @@ def merino_job(
             "--gcs-path",
             "moz-fx-data-prod-external-data/contextual-services/merino-jobs/wikipedia-exports",
             "--gcp-project",
-            "moz-fx-data-shared-prod",
+            "moz-fx-data-shared-nonprod",
```

Comment:
this should stay using the "`prod`" project as that's what we've defined as a storage for this external data.